share/vizd/config/config_testnet.ini

```diff
@@ -66,7 +66,11 @@ inc-shared-file-size = 2G
 # and resizes. The optimal strategy is do checking of the free space, but not very often.
 block-num-check-free-size = 1000 # each 3000 seconds
 
-plugin = chain p2p json_rpc webserver network_broadcast_api witness test_api database_api private_message follow social_network tags account_by_key operation_history account_history block_info raw_block witness_api
+plugin = witness witness_api
+plugin = chain p2p json_rpc webserver network_broadcast_api test_api database_api
+plugin = account_history operation_history
+plugin = committee_api invite_api paid_subscription_api
+plugin = private_message follow social_network tags account_by_key block_info raw_block
```

Comment:
>follow social_network

Are these plugins really used in viz?

Reply:
I don't know about used but exists

Reply:
Remove deprecated and template test:
private_message
follow
social_network
tags
test_api